Cargo.toml

```diff
@@ -285,6 +285,8 @@ getrandom = { git = "https://github.com/madsim-rs/getrandom.git", rev = "e79a7ae
 tokio-stream = { git = "https://github.com/madsim-rs/tokio.git", rev = "fe39bb8e" }
 tokio-retry = { git = "https://github.com/madsim-rs/rust-tokio-retry.git", rev = "95e2fd3" }
 tokio-postgres = { git = "https://github.com/madsim-rs/rust-postgres.git", rev = "ac00d88" }
+sqlx = { git = "https://github.com/KveinAxel/sqlx.git", rev = "f7bc5da" }
+madsim = { git = "https://github.com/madsim-rs/madsim.git", rev = "fb3dc13" }
```

Comment:
Why do we pin a madsim commit and sqlx commit?

Reply:
I guess it's for https://github.com/KveinAxel/sqlx/commit/2635d79197d80b1297f1cda50062fce7d94ce7a4, which looks mainly for adding `async` to all the functions.